src/schema/v2/artwork/artworkContextGrids/__tests__/defaultContext.test.ts

```diff
@@ -71,6 +71,28 @@ describe("Default Context", () => {
     }
   })
 
+  it("Calls the loader class with the correct arguments", async () => {
+    const artistArtworks = [
+      { id: "artwork1", title: "Artwork 1" },
+      { id: "artwork2", title: "Artwork 2" },
+      { id: "artwork3", title: "Artwork 3" },
+    ]
+    context.artistArtworksLoader = jest.fn(() =>
+      Promise.resolve(artistArtworks)
+    )
```

Comment:
Is this block needed or it is already being mocked in the `beforeEach` block [above](https://github.com/artsy/metaphysics/pull/2144/files#diff-da0029ccfd26c7da1ca996217d7e48d7R60)?

Reply:
Good question!  Glad you called this out - I am a little insecure about this blurb and let me know if you have any hints and improvements!  

I added this block here to convert the `artistArtworksLoader` function into a mock function with Jest. Without this change I wasn't able to access the arguments/use the `toHaveBeenCalledWith` matcher and would only have access to the results of the function after it had ran.

Reply:
Ah I see yes you are right it makes total sense. Sorry I didn't notice you added that `jest.fn` part.